src/adapters/IDB.js

```diff
@@ -146,7 +157,20 @@ export default class IDB extends BaseAdapter {
       return Promise.resolve(this);
     }
     return new Promise((resolve, reject) => {
-      const request = indexedDB.open(this.dbname, 1);
+      const version = 1;
+      let request;
+      if(this._options.storage) {
+        try {
+          request = indexedDB.open(this.dbname, {
+            storage: this._options.storage,
+            version: version
+          });
+        } catch(e) {
+          request = indexedDB.open(this.dbname, version);
+        }
+      } else {
+        request = indexedDB.open(this.dbname, version);
+      }
```

Comment:
I think we could simplify this code a bit, especially this `if (this._options.storage)`, with something like `const options = this._options ? {version, ...this._options} : version;`